### PR TITLE
Katacoda is dead. Sad times.

### PR DESCRIPTION
**Pull request by farcaller**

### CODE REVIEW

**Comment by cwilkers** [comment]
Hi @farcaller , thanks for the PR (and for catching this link)
/ok-to-test

So Katacoda is dead, but we have new content going up at Killercoda, and this PR would be a good place for it. I'll add a review with an update to the link if you're willing

File: docs/index.md [hunk 1]
```diff
@@ -5,8 +5,6 @@ This page is provided as the entrypoint to the different topics of this user-gui
 
 ## Try it out
```
Comment (cwilkers) [inline docs/index.md:7]:
```suggestion

Kubevirt on killercoda: <https://killercoda.com/kubevirt>
```

**Review by cwilkers — COMMENTED** [review]
Here's the link

**Comment by farcaller** [comment]
Thanks, link added.

**Comment by cwilkers** [comment]
Thanks!

/approve
/lgtm

**Pull request merged**